### DISCUSSION TITLE
Error in imgui_draw.cpp: Unhandled exception at 0x00007FF6811DB2DD in program.exe: 0xC0000005: Access violation reading location 0xFFFFFFFFFFFFFFD8.

### DISCUSSION THREAD

**Discussion by T1GxR** [ocornut/imgui#7507] [New Users Build/Link/Run issues ONLY!]
The program builds succesfully but after a few seconds i get this unhandled exception which stops the debugging.
This exception is located in imgui_draw.cpp, at the line 3277. This is the part of the code:

>  // Setup fallback character

    const ImWchar fallback_chars[] = { (ImWchar)IM_UNICODE_CODEPOINT_INVALID, (ImWchar)'?', (ImWchar)' ' };
    FallbackGlyph = FindGlyphNoFallback(FallbackChar);
    if (FallbackGlyph == NULL)
    {
        FallbackChar = FindFirstExistingGlyph(this, fallback_chars, IM_ARRAYSIZE(fallback_chars));
        FallbackGlyph = FindGlyphNoFallback(FallbackChar);
        if (FallbackGlyph == NULL)
        {
            FallbackGlyph = &Glyphs.back();
            FallbackChar = (ImWchar)FallbackGlyph->Codepoint;
        }
    }
    
    FallbackAdvanceX = FallbackGlyph->AdvanceX;
    for (int i = 0; i < max_codepoint + 1; i++)
        if (IndexAdvanceX[i] < 0.0f)
            IndexAdvanceX[i] = FallbackAdvanceX;
}

Screenshots:
![Screenshot 2024-04-18 195623](https://github.com/ocornut/imgui/assets/106729571/2c3b9635-25ac-48f8-ab70-48aa39f0cd9d)
![Screenshot 2024-04-18 195840](https://github.com/ocornut/imgui/assets/106729571/c5b7d146-e7c0-411a-b914-b993468b9d6e)

**Comment by ocornut** [comment]
Sorry but #1586
